mev_inspect/schemas/strategy.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional, List
+from enum import Enum
+from pydantic import BaseModel
+from .classified_traces import ClassifiedTrace, Protocol
+
+class StrategyType(Enum):
+    arbitrage = "arbitrage"
+    liquidation = "liquidation"
+
+class Strategy(BaseModel):
```

Comment:
I think better for now to just have a `Liquidation` without a base `Strategy` object.

We can combine them if we find a use for something being either a `Liquidation` or an `Arbitrage`